libbpfgo.h

```diff
@@ -17,6 +17,7 @@
 #include <bpf/bpf.h>
 #include <bpf/libbpf.h>
 #include <linux/bpf.h> // uapi
```

Comment:
libbpfgo should not depend on UAPI afaik, that was the recent change/fix I did for selftests, failing for UAPI dependencies. And, that is exactly the problem here. This type should be declared in a "vmlinux.h" like file and, if thats the case, have different flavors of the type per kernel version (so libbpf relocations can work).

Reply:
fix done for the selftests: https://github.com/aquasecurity/libbpfgo/commit/99ea32d09a1e8ca9ba644a73ac73dcc32c6424f9